CMakeLists.txt

```
@@ -2665,7 +2670,7 @@ endif()
 option(DEBUG_ASSERTIONS_FATAL "Fail if debug become true assertions" OFF)
 if(DEBUG_ASSERTIONS_FATAL)
   target_compile_definitions(mixxx-lib PUBLIC MIXXX_DEBUG_ASSERTIONS_FATAL MIXXX_DEBUG_ASSERTIONS_ENABLED)
-  if(QML)
+  if(QT6)
```

Ah looks like this should probably be

```suggestion
  if(QML)
```